flexget/plugins/urlrewrite_divxatope.py

```diff
@@ -8,24 +8,59 @@
 from flexget.utils import requests
 from flexget.utils.soup import get_soup
 
+from flexget.entry import Entry
+from flexget.utils.search import torrent_availability, normalize_unicode
+
 log = logging.getLogger('divxatope')
 
+CATEGORIES = {
```

Comment:
Shall I change the title to WIP for now or do you want to submit the plugin without category support, if so perhaps remove categories on this PR